tools/kubeblocks_helm/pika-master-slave/dataprotection/backup.sh

```
@@ -0,0 +1,39 @@
+${connect_url} BGSAVE
+
+echo "INFO: wait for saving dump successfully"
+while true; do
+  end_save=$(${connect_url} LASTSAVE)
+  if [ $end_save -ne $last_save ];then
+     break
+  fi
+  sleep 1
+done
+
+cd ${DATA_DIR}
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)